front/src/modules/ui/board/hooks/useUpdateBoardCardIds.ts

```diff
@@ -0,0 +1,86 @@
+import { DropResult } from '@hello-pangea/dnd'; // Atlassian dnd does not support StrictMode from RN 18, so we use a fork @hello-pangea/dnd https://github.com/atlassian/react-beautiful-dnd/issues/2350
+import { useRecoilCallback } from 'recoil';
+
+import { boardCardIdsByColumnIdFamilyState } from '../states/boardCardIdsByColumnIdFamilyState';
+import { boardColumnsState } from '../states/boardColumnsState';
+import { BoardColumnDefinition } from '../types/BoardColumnDefinition';
+
+export function useUpdateBoardCardIds() {
```

Comment:
Is this the right name ? My understanding is that this method handles the card drop only, not "Update board card ids". Correct ?

Reply:
It updates the board card ids but only in recoil, maybe we should specify this ? Or useHandleBoardCardDrop ?